tests/vaults.test.mjs

```diff
@@ -611,7 +612,15 @@ describe('Vaults', async () => {
         from: secondVaulter,
         msgId: vaultId2,
       });
-      paginatedVaultMemory = updatedMemory2;
+      const { memory: updatedMemory3 } = await createVault({
```

Comment:
nit - have this a createVaultedTransfer and assert the controller is returned as well